### PR TITLE
Removed ExceptionAction.CONTINUE_WAIT

### PR DESCRIPTION
**Pull request by pveentjer**
It is not being used.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java [hunk 1]
```diff
@@ -398,9 +398,6 @@ void notifyError(Object error) {
         }
 
         switch (onException(cause)) {
-            case CONTINUE_WAIT:
-                handleContinueWait();
```
Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java]: isn't this method now not used too?

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java]: This code is not used at all.

Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java]: so shouldn't you remove the `handleContinueWait()` method too in this PR?

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java]: It will be fixed any second. Any other comments?

**Comment by jerrinot** [comment]
what's the impact on SPI users? what's a replacement?

**Comment by pveentjer** [comment]
There is no replacement. If you are using it; bad luck. The mechanism is just not supported and provides no value with the operation timeout logic replacement I'm working on.

**Comment by jerrinot** [comment]
couldn't we deprecate it and treat it as no-op? as that's what it is (once the new timeout is in place)
this way you backward maintain compatibility + avoid code complexity (apart from extra enum)

**Comment by pveentjer** [comment]
We are breaking api regarding operation call timeouts anyway. TraceableOperation for example is removed because we don't trace any operations. So why not get this part fixed as well.

In the SPI we can break stuff and this is rarely used.

**Comment by jerrinot** [comment]
:+1: 
I don't have a strong opinion here. Please wait for a 2nd reviewer before merging the PR.

**Comment by mtopolnik** [comment]
:+1:

**Pull request merged**